cmd/launcher/internal/timemachine/timemachine_darwin.go

```diff
@@ -0,0 +1,36 @@
+//go:build darwin
+// +build darwin
+
+package timemachine
```

Comment:
had no idea where to put this 🤷

Reply:
Could see it living in `ee/agent/`, maybe?

Reply:
yea, that seems better